### PR TITLE
Update to 2.1.1

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

**Comment by nedrichards** [comment]
We should also update wxWdigets and the GNOME runtime here.

```
{
            "name": "wxWidgets",
            "rm-configure": true,
            "config-opts": [
                "--with-gtk=3", 
                "--disable-sdltest", 
                "--disable-webkit", 
                "--disable-webview",
                "--enable-debug",
                "--with-cxx=14",
                "--enable-stl",
                "--disable-webviewwebkit"
            ],
            "sources": [{
                    "type": "archive",
                    "url": "https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.4/wxWidgets-3.0.4.tar.bz2",
                    "sha256": "96157f988d261b7368e5340afa1a0cad943768f35929c22841f62c25b17bf7f0"
                },
                {
                    "type": "script",
                    "path": "autogen.sh",
                    "commands": [
                        "cp -p /usr/share/automake-*/config.{sub,guess} .",
                        "autoconf -f -B build/autoconf_prepend-include"
                    ]
                }
            ]
        },
```
That follows the configuration from upstream's snap more closely.

File: net.poedit.Poedit.json [hunk 1]
```diff
@@ -119,9 +119,13 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://github.com/vslavik/poedit/releases/download/v2.0.6-oss/poedit-2.0.6.tar.gz",
-                    "sha256": "a9e9feee6d65440008ae3fe24504681fe390cb41233122580f503a677989b1bc"
-                }
+                    "url": "https://github.com/vslavik/poedit/archive/v2.1.1-oss.tar.gz",
+                    "sha256": "6ae1af449c958a7c517f3307472cfb6b4643db74cfaaa3fc8a59dbb7c8ea51e8"
+                },
+	    		{
```
Comment (nedrichards) [inline net.poedit.Poedit.json]: the alignment is a bit shonky here.

**Comment by nedrichards** [comment]
This needs:
>  "--talk-name=org.freedesktop.secrets",
as well.

**Comment by bilelmoussaoui** [comment]
bot, build

**Comment by vslavik** [comment]
> That follows the configuration from upstream's snap more closely.

Not really, that uses a git submodule referencing a post-3.1.1 snapshot of wx, and building using the submodules references in always best (`flatpak-builder` seems to support submodules…?). In particular, using GTK+3 with wxGTK-3.0 is not a good idea, 3.0 was pretty buggy with it.

**Comment by bilelmoussaoui** [comment]
@vslavik So we are supposed to not build wx but only use the submodule you have on Poedit?

**Comment by vslavik** [comment]
> @vslavik So we are supposed to not build wx but only use the submodule you have on Poedit?

I don't follow. The submodule points to wx's repository and of course has to be built. I'm saying that it's not true that a build using wxGTK-3.0.4 matches the snap (meaningfully) more closely, as the latter uses post-3.1.1 wxGTK. 

To be perfectly clear: using wxGTK-3.1.1 is significantly better than using 3.0.4 and using the revision in Poedit's `deps/wx` submodule is (less significantly) better than 3.1.1.

**Comment by bilelmoussaoui** [comment]
alright, will switch to 3.1.1 !

**Comment by bilelmoussaoui** [comment]
bot, build

**Comment by nedrichards** [comment]
@bilelmoussaoui the appdata patch failed to apply: https://flathub.org/builds/#/builders/2/builds/4207

**Comment by bilelmoussaoui** [comment]
Closing in favor of #6

**Pull request closed without merging**